HIP/hip-0000-long-term-scheduled-transactions.md

```diff
@@ -0,0 +1,399 @@
+
+_When `ScheduleCreate` is executed, apply throttles to the future second when it's transaction will execute. In other words, make sure throttles will never be exceeded in any given second in the future._
+
+### Backwards Compatible
+
+- The changes to the existing Scheduled Transactions implementation should be additive.
+- Our changes should not break any existing functionality. 
+
+_By default, everything should continue to function as it does today._
+
+### Fees
+
+- It will be relatively cheap to "fill up" Scheduled Transactions such that you block others from using them.
+- Some transactions, like `ContractCall`, would be particularly easy and possibly profitable to block.
+  - For example, an attacker could schedule a large number of very high `gasLimit` `ContractCall` transactions and cheaply use up all the available gas for a future block of time.
+    - The attacker doesn't actually need to have enough HBAR in their account to cover the gas.
+    - When the future date comes around all the transactions will fail - but that's not a problem for the attacker, they prevented others from using the feature.
+
+_Charge a larger fee for `ScheduleCreate` transactions containing certain types of transactions._
+
+### Types of transactions
+
+- The Hedera Council does not use **all** types of transactions, only a subset.
+- It would be a significant win to support scheduling smart contract operations.
+
+_To limit the testing/attack surface, only add the types of transactions the Hedera Council needs and smart contracts require_
+
+### Transaction Receipts and Transaction IDs
+
+- Now that Scheduled Transactions can execute autonomously in the future, there needs to be a way to fetch the transaction receipt for them outside the normal mechanism.
+- A user should not need to plan to fetch the transaction receipts within 3 minutes of their Scheduled Transaction executing
+
+_Data from the transaction receipt is available in the `entity_id` and other fields in the mirror node transactions API._
+
+- We need a transaction ID to fetch from the transaction api to get receipts.
+- The transaction ID of the transaction executed by a Scheduled Transaction is currently just the `ScheduleCreate` transaction ID with the `scheduled` flag set to true.
+
+_The transaction ID of the child can be found deterministically at `ScheduleCreate` time_
+
+_Users can also use the `executed_timestamp` in the mirror node schedule API to find child transactions via the mirror node transactions API._
+
+### Corner cases
+
+- An account changes keys after it has signed a Scheduled Transaction. 
+  - Scheduled Transactions will need to be re-signed with the new keys for them to succeed in this case.
+- Account deletion.
```

Comment:
Could be generalized to "Invalidation of pending scheduled transaction"...even if a transaction was valid at `ScheduleCreate`, any number of things could happen (e.g., an account is deleted) that will cause it to fail at execution time.

Reply:
reworded :-)